source/guide_laravel.rst

```diff
@@ -0,0 +1,151 @@
+.. highlight:: console
+
+.. author:: Christian Kantelberg <uberlab@mailbox.org>
+
+.. tag:: lang-php
+.. tag:: web
+
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/laravel-logo.png
+      :align: center
+
+##########
+Laravel
+##########
+
+.. tag_list::
+
+Laravel is a web application framework with expressive, elegant syntax. We’ve already laid the foundation — freeing you to create without sweating the small things.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`Domains <web-domains>`
+
+
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+Your web domain needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Fresh Installation
+======================
+
+``cd`` to your user dir, then get Laravel installer via Composer. Finally create a new project using the installer.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust html]$ composer global require "laravel/installer"
+ [isabell@stardust html]$ laravel new <projectname>
+ Application ready! Build something amazing.
+ [isabell@stardust ~]$
+ 
+ Clone your project from github
+======================
+
+If you like to have your own project just clone it into your personal folder /var/www/virtual/$USER/
+Normaly the vendor files should not be included we get them via composer
+
+::
+
+ [isabell@stardust ~]$ cd <projectname>
+ [isabell@stardust <projectname>]$ composer install
+
+
+Set a symbolic link
+======================
+
+The Laravel project is now in your user folder. But Uberspace serves only the /html folder. 
+The accessible files from Laravel are contained in the project/public folder. Therefore we build a symbilic link.
+You can wether built it for a domain already listed on your uberspace or your default domain $user.uber.space. 
+
+First we rename the old html folder then we create the symbolic link. If you browse now $USER.uber.space you get redirected to /var/www/virtual/$USER/<PROJEKTNAME>/public where your index.php start the application up.
+
+::
+
+ [isabell@stardust ~]$ mv html/ htmlbackup/
```

Comment:
`~/html` is just a symlink, moving it does not do anything. Are you sure the prompt should specify `~` as the working directory here?